rasa/core/tracker_store.py

```diff
@@ -118,16 +118,32 @@ def load_tracker_from_module_string(
         """
         custom_tracker = None
         try:
-            custom_tracker = class_from_module_path(store.type)
+            custom_tracker = common.class_from_module_path(store.type)
         except (AttributeError, ImportError):
             warnings.warn(
                 f"Store type '{store.type}' not found. "
-                "Using InMemoryTrackerStore instead"
+                f"Using InMemoryTrackerStore instead."
             )
 
         if custom_tracker:
+            init_args = common.arguments_of(custom_tracker.__init__)
+            if "url" in init_args and "host" not in init_args:
+                warnings.warn(
+                    "The `url` initialization argument for custom tracker stores is deprecated. Your "
+                    "custom tracker store should take a `host` argument in ``__init__()`` instead.",
+                    DeprecationWarning,
+                )
+                return custom_tracker(
```

Comment:
a bit less duplicate:
```suggestion
                store.kwargs["url"] = store.url
            else:
                store.kwargs["host]= store.url
            return custom_tracker(
                domain=domain,
                event_broker=event_broker,
                **store.kwargs,
                
```

Reply:
very nice!